src/components/Profile/components/PersonalInfoList/index.js

```diff
@@ -49,7 +49,7 @@ const PersonalInfoList = ({ myProf, profile, isOnline, createSnackbar }) => {
   const [showMailCombo, setShowMailCombo] = useState(false);
   const isStudent = profile.PersonType?.includes('stu');
   const isFacStaff = profile.PersonType?.includes('fac');
-  const isAlumni = profile.PersonType?.includes('alu');
+  const isAlumni = true;
```

Comment:
Don't merge your personal tests :)

Reply:
I think this was on me. Most likely an error from switching between 2 VM's and 2 local environments. Might have accidentally staged it. I'll tear it back out. Good catch!